afs/afs-ws/afs-ws-storage/src/main/java/com/powsybl/afs/ws/storage/NodeEventClient.java

```diff
@@ -27,9 +36,17 @@
 
     private final WeakListenerList<AppStorageListener> listeners;
 
-    public NodeEventClient(String fileSystemName, WeakListenerList<AppStorageListener> listeners) {
+    private final RemoteServiceConfig config;
+
+    private int reconnectionInterval;
+
+    private static int counter = 0;
```

Comment:
Should it really be a static variable? Why this counter is shared by all NodeEventClient instances?

Reply:
The counter attribute is replaced by a local variable in method onClose()